packages/blockly-field-date/package.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "@blockly/field-date",
+    "version": "3.20200304.0",
+    "description": "A Blockly date picker field that uses the Google Closure date picker.",
+    "scripts": {
+        "build": "gulp build"
```

Comment:
Are there any extra steps for publishing? If so, can we add a prepublishOnly script? It would be nice if in all the repositories all we had to do was login and run npm publish?